alerting/alerter-war/src/main/java/org/hawkular/metrics/alerter/ConditionManager.java

```diff
@@ -489,15 +495,17 @@ private void evaluateEach(String target, Map<String, Map<String, List<? extends
             }
         }
 
-        private void evaluate(String target, Map<String, BucketPoint> queryResults, ConditionEvaluator evaluator) {
+        private void evaluate(String target, Map<String, BucketPoint> queryResults, ConditionExpression expression) {
+            ConditionEvaluator evaluator = expression.getEvaluator();
             Map<String, String> preparedCondition = null;
             try {
                 preparedCondition = evaluator.prepare(queryResults);
             } catch (Exception e) {
                 log.debugf("Could not prepare evaluation, Skipping due to [%s]", e.getMessage());
             }
 
-            if (evaluator.evaluate()) {
+            boolean evalResult = evaluator.evaluate();
+            if (!isQuiet("_ALL_", expression.getQuietCount(), evalResult)) {
```

Comment:
Minor one: I wonder if instead to treat "_ALL_" as a metricName, if it is not better just to pass a boolean flag to indicate all/each.